Sources/SafeDICore/Models/Initializer.swift

```diff
@@ -177,7 +177,7 @@ public struct Initializer: Codable, Hashable {
         try arguments.reduce(into: [(dependency: Dependency, argument: Argument)]()) { partialResult, argument in
             guard let dependency = dependencies.first(where: {
                 $0.property.label == argument.innerLabel
-                && $0.property.typeDescription == argument.typeDescription
+                && $0.property.typeDescription.isEqualToFunctionArgument(argument.typeDescription)
```

Comment:
Any reason why you don't override `==`?

Reply:
because these things are _not_ equal. They're only equal in this specific property <-> function argument comparison context.

Reply:
Ahhh gotcha